app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AdminController < ApplicationController
+
+    def create
```

Comment:
missing contents for the create method, this will return an error when clicking submit in the form
missing `def show` as well

please do these corrections in the major revision instead